strings/french.json

```diff
@@ -183,7 +191,7 @@
   "This action {strong_}cannot be undone{_strong}.": "Cette action {strong_}est irréversible{_strong}.",
   "Delete channel": "Supprimer le salon",
   "Channel description": "Description du salon",
-  "The description is limited to 280 characters": "La description est limitée à 280 caractères",
+  "This field is limited to characters": "Ce champ est limitée à caractères",
```

Comment:
Ce champ est limitée à 280 caractères

Reply:
I stored that constant 280 in the chatroom contract settings and made it mutable.
So I wanted to make a variant translation regarding the limitation.
I know we can make a variant translation like `"Joined {title}"`, but actually I couldn't find the way to make a validation for this variant translation. That's the reason why I removed that 280 in translation.
@pieer, @taoeffect. Do you know the way how to create a validation for the variant translation?
Or is it unnecessary to make the limitation mutable?

Reply:
I mean the translation is useless since the sentence make no sense without the "280"

Reply:
@Silver-IT - @pieer is right, "This field is limited to characters" is not a proper English sentence. It is grammatically incorrect.

You've got two options:

1. Restore the original sentence.
2. Change it and the code so that it uses a variable, and therefore reads: `"This field is limited to {numChars} characters"`